app/models/hyrax/file_metadata.rb

```diff
@@ -3,7 +3,7 @@
 module Hyrax
   class FileMetadata < Valkyrie::Resource
     attribute :file_identifiers, ::Valkyrie::Types::Set # id of the file stored by the storage adapter
-    attribute :alternate_id, ::Valkyrie::Types::Set # id of the Hydra::PCDM::File which holds metadata and the file in ActiveFedora
+    attribute :alternate_ids, Valkyrie::Types::Set.of(Valkyrie::Types::ID) # id of the Hydra::PCDM::File which holds metadata and the file in ActiveFedora
```

Comment:
Required to be alternate_ids which is a standard defined Valkyrie attribute used by the query service.

Reply:
oh! glad we caught this